packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -357,7 +389,10 @@ export class StripeWebhookHandler extends StripeHandler {
       return;
     }
     const invoice = event.data.object as Stripe.Invoice;
-    if (!(await this.stripeHelper.invoicePayableWithPaypal(invoice))) {
+    if (
+      !(await this.stripeHelper.invoicePayableWithPaypal(invoice)) ||
+      invoice.status !== 'draft'
```

Comment:
Maybe swap the conditions so that it can short-circuit on the easier one.

Reply:
Yea, I considered that, but we don't accidentally finalize this very often, so it didn't seem very impactful.